pkg/workload/histogram/exporter/openmetrics_exporter.go

```diff
@@ -0,0 +1,70 @@
+// Copyright 2024 The Cockroach Authors.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package exporter
+
+import (
+	"bufio"
+	"fmt"
+	"io"
+	"strings"
+	"time"
+
+	"github.com/codahale/hdrhistogram"
+	"github.com/prometheus/common/expfmt"
+)
+
+// OpenmetricsExporter exports metrics in openmetrics format
+type OpenmetricsExporter struct {
+	writer *bufio.Writer
+	Labels *map[string]string
+}
+
+func (o *OpenmetricsExporter) Validate(filePath string) error {
+	if strings.HasSuffix(filePath, ".json") {
+		return fmt.Errorf("file path must not end with .json")
+	}
+	return nil
+}
+
+func (o *OpenmetricsExporter) Init(w *io.Writer) error {
```

Comment:
Should we drop `error` from the signature? Neither implementation can yield an error.